src/main/resources/assets/caverns_and_chasms/lang/ru_ru.json

```
@@ -0,0 +1,810 @@
+  "block.caverns_and_chasms.mossy_cobblestone_brick_wall": "Ограда из замшелого булыжного кирпича",
+  "block.caverns_and_chasms.mossy_cobblestone_brick_slab": "Плита из замшелого булыжного кирпича",
+  "block.caverns_and_chasms.cobblestone_tiles": "Булыжная мозаика",
+  "block.caverns_and_chasms.cobblestone_tile_stairs": "Ступеньки из булыжной мозаики",
+  "block.caverns_and_chasms.cobblestone_tile_wall": "Ограда из булыжной мозаики",
+  "block.caverns_and_chasms.cobblestone_tile_slab": "Плита из булыжной мозаики",
+  "block.caverns_and_chasms.mossy_cobblestone_tiles": "Замшелая булыжная мозаика",
+  "block.caverns_and_chasms.mossy_cobblestone_tile_stairs": "Ступеньки из замшелой булыжной мозаики",
+  "block.caverns_and_chasms.mossy_cobblestone_tile_wall": "Ограда из замшелой булыжной мозаики",
+  "block.caverns_and_chasms.mossy_cobblestone_tile_slab": "Плита из замшелой булыжной мозаики",
+
+  "block.caverns_and_chasms.chiseled_deepslate_bricks": "Резные глубинносланцевые кирпичи",
+  "block.caverns_and_chasms.cobbled_deepslate_bricks": "Колотые глубинносланцевые кирпичи",
+  "block.caverns_and_chasms.cobbled_deepslate_brick_stairs": "Ступеньки из колотого глубинносланцевого кирпича",
+  "block.caverns_and_chasms.cobbled_deepslate_brick_wall": "Ограда из колотого глубинносланцевого кирпича",
+  "block.caverns_and_chasms.cobbled_deepslate_brick_slab": "Плита из колотого глубинносланцевого кирпича",
+  "block.caverns_and_chasms.cobbled_deepslate_tiles": "Колотый глубинносланцевый плитняк",
+  "block.caverns_and_chasms.cobbled_deepslate_tile_stairs": "Ступеньки из колотого глубинносланцевого плитняка",
+  "block.caverns_and_chasms.cobbled_deepslate_tile_wall": "Ограда из колотого глубинносланцевого плитняка",
+  "block.caverns_and_chasms.cobbled_deepslate_tile_slab": "Плита из колотого глубинносланцевого плитняка",
+
+  "block.caverns_and_chasms.amethyst": "Аметист",
+  "block.caverns_and_chasms.cut_amethyst": "Пиленый аметист",
+  "block.caverns_and_chasms.cut_amethyst_bricks": "Пиленые аметистовые кирпичи",
+  "block.caverns_and_chasms.cut_amethyst_brick_slab": "Плита из пиленого аметистового кирпича",
+  "block.caverns_and_chasms.cut_amethyst_brick_stairs": "Ступеньки из пиленого аметистового кирпича",
+  "block.caverns_and_chasms.cut_amethyst_brick_wall": "Ограда из пиленого аметистового кирпича",
+
+  "block.caverns_and_chasms.sugilite": "Сугилит",
+  "block.caverns_and_chasms.sugilite_stairs": "Сугилитовые ступеньки",
+  "block.caverns_and_chasms.sugilite_wall": "Сугилитовая ограда",
+  "block.caverns_and_chasms.sugilite_slab": "Сугилитовая плита",
+  "block.caverns_and_chasms.polished_sugilite": "Полированный сугилит",
+  "block.caverns_and_chasms.polished_sugilite_stairs": "Ступеньки из полированного сугилита",
+  "block.caverns_and_chasms.polished_sugilite_slab": "Плита из полированного сугилита",
+
+  "block.caverns_and_chasms.dripstone_shingles": "Натёчнокаменная черепица",

```

Как насчёт использования `-`, как это сделано в переводе полированного-чернитных кирпичей?
```suggestion
  "block.caverns_and_chasms.dripstone_shingles": "Натёчно-каменная черепица",
```

---

"Глубинносланцевый плитняк". Всё? Всё.

---

Да, всё же это ближе.